Makefile.dune

```diff
@@ -80,8 +80,12 @@ quickopt: voboot
 test-suite: voboot
 	dune runtest --no-buffer $(DUNEOPT)
 
-refman-html: voboot
-	dune build @refman-html
+coq-refman.install: voboot
+	dune build $(DUNEOPT) @refman-html
+	dune build $(DUNEOPT) tools/gen_install.exe
+	$(BUILD_CONTEXT)/tools/gen_install.exe $(BUILD_CONTEXT)/doc/sphinx_build/html doc $@
+
+refman-html: coq-refman.install
```

Comment:
I'm unsure of the benefit having a new target in `Makefile.dune` brings. You could copy-paste the content of this target here, couldn't you?

Reply:
The rationale is that we are overriding the default `coq-refman.install` target which happens to be a true file, so IMHO it is better to declare and have the file properly rebuilt, we could even add the right dependencies using a glob.

Anyways it seems that proper support in Dune is coming in 1.8 [pretty soon] so hopefully this hack will be very short-lived.